Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Cancel.hs

```diff
@@ -82,7 +88,10 @@ cancel bookingId _ req = do
     throwError $ RideInvalidStatus "Cannot cancel this ride"
   when (booking.status == SRB.NEW) $ throwError (BookingInvalidStatus "NEW")
   bppBookingId <- fromMaybeM (BookingFieldNotPresent "bppBookingId") booking.bppBookingId
-  cancellationReason <- buildBookingCancelationReason
+  ride <- Esq.runInReplica $ QR.findActiveByRBId booking.id >>= fromMaybeM (RideDoesNotExist $ "BookingId: " <> booking.id.getId)
+  currLocation <- RLOC.getDriverLoc ride.id personId
+  finalDis <- getDistanceBetweenPoints booking.merchantId (getCoordinates currLocation) (getCoordinates booking.fromLocation)
```

Comment:
Rename variable from `finalDes` to something proper.

Reply:
pushed new changes.